pom.xml

```diff
@@ -34,11 +34,17 @@
 
     <scm>
         <connection>scm:git:git://github.com/confluentinc/kafka-rest.git</connection>
-        <developerConnection>scm:git:git@github.com:confluentinc/kafka-rest.git</developerConnection>
+        <developerConnection>scm:git:git@github.com:confluentinc/kafka-rest.git
```

Comment:
really minor style nit, i'm guessing this happened due to editor enforcing line length limits, but either the entire thing should be on one line or the value should be on a separate line from opening and closing tags.